src/Compilers/CSharp/Test/Semantic/Diagnostics/OperationAnalyzerTests.cs

```diff
@@ -748,7 +748,8 @@ enum E
                 Diagnostic(SeventeenTestAnalyzer.SeventeenDescriptor.Id, "17").WithLocation(4, 40),
                 Diagnostic(SeventeenTestAnalyzer.SeventeenDescriptor.Id, "17").WithLocation(9, 21),
                 Diagnostic(SeventeenTestAnalyzer.SeventeenDescriptor.Id, "17").WithLocation(14, 16),
-                Diagnostic(SeventeenTestAnalyzer.SeventeenDescriptor.Id, "17").WithLocation(24, 9)
+                Diagnostic(SeventeenTestAnalyzer.SeventeenDescriptor.Id, "17").WithLocation(24, 9),
+                Diagnostic(SeventeenTestAnalyzer.SeventeenDescriptor.Id, "M0()").WithLocation(16, 9)
```

Comment:
It's at least defensible. The pre-walker strategy didn't find the argument-supplied-by-default because it only found operations that had corresponding syntax. One can make a case that there should be no semantic difference between passing 17 explicitly and passing it implicitly.

I'm too tired to trust my own judgment, but I believe that the new behavior is correct.

I tried the change in the walker myself a few days ago, got as far as noticing the test failure, and abandoned the change. Perseverance furthers.